packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, variables } from '@trezor/components';
+import { Translation } from './Translation';
+import { Account, CoinjoinSession, WalletParams } from '@suite-common/wallet-types';
+import { CountdownTimer } from './Timer';
+import { useActions } from '@suite-hooks/useActions';
+import * as routerActions from '@suite-actions/routerActions';
+import { useSelector } from '@suite-hooks/useSelector';
+
+const SPACING = 6;
+
+const Container = styled.div`
+    display: flex;
+    align-items: center;
+    height: 28px;
+    padding: 0 ${SPACING}px;
+    background: ${({ theme }) => theme.BG_WHITE};
+    border-bottom: 1px solid ${({ theme }) => theme.STROKE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const ProgressPie = styled.div<{ progress: number }>`
+    width: 16px;
+    height: 16px;
+    margin-right: ${SPACING}px;
+    border-radius: 50%;
+    background: ${({ theme, progress }) =>
+        `conic-gradient(${theme.BG_GREEN} ${3.6 * progress}deg, ${theme.STROKE_GREY} 0)`};
+`;
+
+const StatusText = styled.span`
+    color: ${({ theme }) => theme.TYPE_GREEN};
+`;
+
+const NextRoundNote = styled.span`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const Separator = styled.span`
+    margin: 0 ${SPACING / 2}px;
+`;
+
+const ViewButton = styled(Button)`
+    height: 20px;
+    margin-left: auto;
+`;
+
+enum MockCoinjoinPhase {
+    Starting,
+    Ongoing,
+    Error,
+}
+
+const STATUS_MESSAGES = {
+    [MockCoinjoinPhase.Starting]: 'Collecting inputs',
+    [MockCoinjoinPhase.Ongoing]: 'Joining coins',
+    [MockCoinjoinPhase.Error]: 'U poor now',
+};
+
+interface CoinjoinStatusBarProps {
+    accountKey: string;
+    session?: CoinjoinSession;
+}
+
+export const CoinjoinStatusBar = ({ accountKey, session }: CoinjoinStatusBarProps) => {
+    const accounts = useSelector(state => state.wallet.accounts);
+    const routerParams = useSelector(state => state.router.params) as WalletParams;
+
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    const phase = MockCoinjoinPhase.Ongoing;
+    const progress = 36; //  current_phase / (total_phases / 100)
+
+    const { symbol, index, accountType } = accounts.find(
+        ({ key }) => key === accountKey,
+    ) as Account;
```

Comment:
Better to handle undefined state than asserting Account type
<img width="604" alt="image" src="https://user-images.githubusercontent.com/3729633/192995826-e7bedecb-ceb3-4419-9239-28c61508806a.png">

Reply:
553294c65ebc0f98cdd168c80c14542ad398e998